package.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0"?>
 <package format="2">
   <name>astra_camera</name>
-  <version>0.2.0</version>
+  <version>2.0.0</version>
   <description>Drivers for Orbbec Astra Devices. </description>
   <author email="liuhua@orbbec.com">Tim Liu</author>
   <license>BSD</license>
 
-  <maintainer email="liuhua@orbbec.com">Tim Liu</maintainer>
```

Comment:
I think that Tim Liu should stay credited as an author, no?

Reply:
He is, above.  He was both author and maintainer, so I just switched maintainer.

Reply:
`>_<` too used to the format. Can you please move the author **after** the licence and the maintainer **before**  as the package.xml format specifies